### PR TITLE
Removed OpenPilot reference in About menu

### PR DESCRIPTION
**Pull request by paul-jewell**
This is a clean version of pull request #20, which I will close shortly.

### CODE REVIEW

**Comment by parched** [comment]
Can we not use the define?

**Comment by paul-jewell** [comment]
Good point - I'll have a look...

**Comment by paul-jewell** [comment]
Now using the ORG_BIG_NAME cascading from Makefile.

File: ground/openpilotgcs/src/plugins/coreplugin/mainwindow.cpp [hunk 1]
```diff
@@ -94,6 +94,8 @@ using namespace Core::Internal;
 
 static const char *uriListMimeFormatC = "text/uri-list";
 
+const char *ORG_NAME(ORG_BIG_NAME);
+
```
Comment (parched) [inline ground/openpilotgcs/src/plugins/coreplugin/mainwindow.cpp]: Is needed/good practice?

Comment (filnet) [inline ground/openpilotgcs/src/plugins/coreplugin/mainwindow.cpp]: i guess compilers do that for you...

Comment (paul-jewell) [inline ground/openpilotgcs/src/plugins/coreplugin/mainwindow.cpp]: umm - no! Thanks for pointing that out

**Comment by paul-jewell** [comment]
Replaced by PR#35

**Pull request closed without merging**